bentoml/yatai/deployment/azure_function/templates.py

```diff
@@ -0,0 +1,127 @@
+AZURE_API_FUNCTION_JSON = """\
+{{
+  "scriptFile": "__init__.py",
+  "bindings": [
+    {{
+      "authLevel": "{function_auth_level}",
+      "type": "httpTrigger",
+      "direction": "in",
+      "name": "req",
+      "methods": [
+        "get",
+        "post",
+        "put",
+        "delete",
+        "patch"
+      ],
+      "route": "{{*route}}"
+    }},
+    {{
+      "type": "http",
+      "direction": "out",
+      "name": "$return"
+    }}
+  ]
+}}
+"""
+
+AZURE_API_INIT_PY = """\
+import azure.functions as func
+
+from bentoml.server import BentoAPIServer
+from bentoml import load
+
+from __app__.{bento_name} import load
```

Comment:
Is there a way to avoid templating Python code?  We should avoid it as much as possible

I think something like `svc = load(DIR)` should work right?

Reply:
I am not so sure with Azure's import behavior https://docs.microsoft.com/en-us/azure/azure-functions/functions-reference-python#import-behavior

Reply:
it should work